bsp/src/mill/bsp/MillBspLogger.scala

```diff
@@ -52,7 +52,7 @@ class MillBspLogger(client: BuildClient, taskId: Int, logger: Logger)
 
   override def debug(s: String): Unit = {
     super.debug(s)
-    client.onBuildShowMessage(new ShowMessageParams(MessageType.LOG, s))
+    client.onBuildLogMessage(new LogMessageParams(MessageType.LOG, s))
```

Comment:
According to the other comment (https://github.com/com-lihaoyi/mill/pull/2006/files#r954194830), we should put this behind a guard.

```suggestion
    if(debugEnabled) {
      client.onBuildLogMessage(new LogMessageParams(MessageType.LOG, s))
    }
```

Reply:
I moved this guard in the other PR.